Setting up Hello-Microverse project with Html&Css

tsigewossne opened the pull request:
## In This project I have created:
### Basic Files
- A .html file that includes a header with the text "Hello Microverse!". which uses the h1 tag for that.
- A . CSS file that includes one class with styles for the header (Aqua color).
- A test. md file with the text "This file should be ignored by git".
### Linters Configuration
### A descriptive README file
### .gitignore file to include files that should be ignored from the repository

tsigewossne committed:
creating an html file with Heading1 text

tsigewossne committed:
Adding abackground-color stylsheet for the text in html file

tsigewossne committed:
creating a text file test.md

tsigewossne committed:
adding a new git ignore file

tsigewossne committed:
Modifying .gitignore file by adding node_modules/ and test.md files

tsigewossne committed:
Adding linters configuration file

tsigewossne committed:
Modified style with required indentations

tsigewossne committed:
configuration of linters for Html and Css

tsigewossne committed:
creating for descriptive Readme file

Shedrack-Sunday commented on README.md:
```diff
@@ -0,0 +1,98 @@
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+<!-- PROJECT DESCRIPTION -->
+
```
- [ ] The header section here is complete. you're missing a key section which is the LICENSE file. Kindly add it up here, at the root of your project, and also the path at the bottom as it's done on the template.
- The file is very important to show ownership

## Header:
![image](https://user-images.githubusercontent.com/50990925/235437017-1bcc4828-a304-4fc4-8bcf-d63feb7ce926.png)

## Path:
![image](https://user-images.githubusercontent.com/50990925/235437050-2bed1ca1-fb28-4bbb-bfda-fd9328225b09.png)

Shedrack-Sunday commented on test.md:
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
- [ ] This file should be ideally ignored by git. As such, it should not be committed to your project. Kindly re-add the test file in `.gitignore`, and make sure this does not show, as that's the essence of the `.gitignore` file

Shedrack-Sunday reviewed (changes requested):
Hello @tsigewossne       👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing some of the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 
![](https://user-images.githubusercontent.com/80612925/157397398-00904e26-60ec-46cd-9510-240661c250ef.gif)

## Highlights 
- Nice setup of linters✔️  
- Good job with using GitHub features  ✔️  
  
# Required Changes  🌵 

- [ ] There are a few things you need to take note of, you have a great project as expected 👍 

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

tsigewossne committed:
Updated readme.md file by adding License of the project as it is MIT licensed

tsigewossne committed:
Updated readme.md file by adding License of the project as it is MIT licensed

tsigewossne committed:
Delete test.md

test.md file created before must be manually deleted in order to be ignored by the git because it is included in a .gitignore file

tsigewossne commented:
@Shedrack-Sunday Thank you for your supportive comments and suggestions. I have made the following changes 

- Add a key section of the License on the README.md file
- Remove the test.md file manually and add it to .gitignore Again so that it is ignored by the repository

Gambit142 reviewed (approved):
Hi @tsigewossne 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️
- Commit message is capitalized and in imperative tense ✔️ 
- It has HTML and CSS files ✔️ 
- It has **_node_module_** and _**test.md**_ git ignored ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
